.travis.mk

```diff
@@ -317,13 +315,13 @@ deps_osx:
 	# try to install the packages either upgrade it to avoid of fails
 	# if the package already exists with the previous version
 	brew install --force ${OSX_PKGS} || brew upgrade ${OSX_PKGS}
-	pip install --force-reinstall -r test-run/requirements.txt
+	pip3 install --user --force-reinstall -r test-run/requirements.txt
 
 deps_osx_github_actions:
 	# try to install the packages either upgrade it to avoid of fails
 	# if the package already exists with the previous version
-	brew install --force ${OSX_PKGS_MIN} || brew upgrade ${OSX_PKGS_MIN}
-	pip install --force-reinstall -r test-run/requirements.txt
+	brew install --force ${OSX_PKGS} || brew upgrade ${OSX_PKGS}
+	pip3 install --force-reinstall -r test-run/requirements.txt
```

Comment:
As I see, `--user` is not needed here? Maybe the problem, that you describe in the commit messsage, disappears?

(The same for static build below.)

Reply:
removed